### PR TITLE
add busy loaders

### PR DESCRIPTION
**Pull request by 0xF013**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-add-busy-loaders.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-add-busy-loaders.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/public/templates/work_packages.list.html [hunk 1]
```diff
@@ -85,8 +85,8 @@
 <back-url></back-url>
 
 
-<div class="work-packages--split-view">
-  <div class="work-packages--list" cg-busy="[settingUpPage,refreshWorkPackages]">
+<div class="work-packages--split-view" cg-busy="[settingUpPage,refreshWorkPackages]">
```
Comment (myabc) [inline frontend/public/templates/work_packages.list.html:88]: @0xF013 FYI, `cg-busy` will add a `position: relative` to the div in question.

**Comment by myabc** [comment]
This is definitely better. The behaviour I'd like to see though is that the detail pane opens immediately, and that a loading indicator is shown within the detail pane itself.

@ulferts should we go ahead and merge now?

**Comment by 0xF013** [comment]
@myabc I think we'd need an intermediate state for that. As of now there is some feedback for the user.

**Pull request merged**